index.bs

```diff
@@ -1461,32 +1461,15 @@ This [=internal method=] accepts no arguments.
 
 [=[RPS]=] use this method to determine whether they can create a new credential using a [=user verification|user-verifying=] [=platform authenticator=].
 Upon invocation, the [=client=] employs a platform-specific procedure to discover available [=user verification|user-verifying=] [=platform authenticators=].
-If successful, the [=client=] then assesses whether the user is willing to create a credential using one of the available [=user verification|user-verifying=] [=platform authenticators=].
-This assessment may include various factors, such as:
-    - Whether the user is running in private or incognito mode.
-    - Whether the user has configured the [=client=] to not create such credentials.
-    - Whether the user has previously expressed an unwillingness to create a new credential for this [=[RP]=],
-        either through configuration or by declining a user interface prompt.
-    - The user's explicitly stated intentions, determined through user interaction.
-If this assessment is affirmative, the promise is resolved with the value of [TRUE].
+If any are discovered, the promise is resolved with the value of [TRUE].
 Otherwise, the promise is resolved with the value of [FALSE].
 Based on the result, the [=[RP]=] can take further actions to guide the user to create a credential.
 
 This method has no arguments and returns a Boolean value.
 
-If the promise will return [FALSE],
-the [=client=] SHOULD wait a fixed period of time from the invocation of the method before returning [FALSE].
-This is done so that callers cannot distinguish between
-the case where the user was unwilling to create a credential using one of the available [=user verification|user-verifying=] [=platform authenticators=] and
-the case where no [=user verification|user-verifying=] [=platform authenticator=] exists.
-Trying to make these cases indistinguishable is done in an attempt to not provide additional information that could be used for fingerprinting.
-A timeout value on the order of 10 minutes is recommended;
-this is enough time for successful user interactions to be performed
-but short enough that the dangling promise will still be resolved in a reasonably timely fashion.
-
 <pre class="idl">
     partial interface PublicKeyCredential {
-        static Promise < boolean > isUserVerifyingPlatformAuthenticatorAvailable();
+        static Promise&lt;boolean&gt; isUserVerifyingPlatformAuthenticatorAvailable();
```

Comment:
using literal angle brackets "< ... >"  in `<pre>` does not seem to be an issue in bikeshed, eg our use of `sequence<...>` all over the place, so perhaps the `&lt` and `&gt` should be literal angle brackets?  I'm ok with removing the gratuitous whitespace.

Reply:
ah, perhaps an issue here is the use of `<pre>` block rather than `<xmp>`, see https://tabatkins.github.io/bikeshed/#xmp

see also #943

Reply:
`pre` got changed to `xmp` throughout so I've switched to `<` and `>` here. Thanks.